edx_rest_framework_extensions/authn/jwt/cookies.py

```diff
@@ -0,0 +1,41 @@
+"""
+JWT Authentication cookie utilities.
+"""
+
+from django.conf import settings
+
+
+def jwt_cookie_name():
+    """
+    Returns the JWT cookie name from either the JWT_AUTH setting, or the
+    default if the setting is not set.
+    """
+    value = settings.JWT_AUTH.get('JWT_AUTH_COOKIE')
```

Comment:
@nasthagiri: Generally I would set up constants for strings, but I think you wanted these strings directly in these methods as implemented.   Since they are only used once and encapsulated, I think this is fine and reads cleanly.

Reply:
agree.